txstratum/cli.py

```diff
@@ -66,29 +72,52 @@ def execute(args: Namespace) -> None:
     loop.run_until_complete(backend.start())
     loop.run_until_complete(manager.start())
     server = loop.run_until_complete(loop.create_server(manager, '0.0.0.0', args.stratum_port))
+    tx_filters: List[TXFilter] = []
+    toiclient: Optional[TOIAsyncClient] = None
+    banned_tx_ids: Set[bytes] = set()
+    banned_addrs: Set[str] = set()
 
     if args.prometheus:
         from txstratum.prometheus import PrometheusExporter
         metrics = PrometheusExporter(manager, args.prometheus)
         metrics.start()
 
-    api_app = App(manager, max_tx_weight=args.max_tx_weight, max_timestamp_delta=args.max_timestamp_delta,
-                  tx_timeout=args.tx_timeout, fix_invalid_timestamp=args.fix_invalid_timestamp,
-                  only_standard_script=not args.allow_non_standard_script)
-    logger.info('API Configuration', max_tx_weight=api_app.max_tx_weight, tx_timeout=api_app.tx_timeout,
-                max_timestamp_delta=api_app.max_timestamp_delta, fix_invalid_timestamp=api_app.fix_invalid_timestamp,
-                only_standard_script=api_app.only_standard_script)
-
     if args.ban_tx_ids:
         fp = open(args.ban_tx_ids, 'r')
         for line in fp:
             line = line.strip()
             if not line:
                 continue
             logger.info('Added to banned tx ids', txid=line)
-            api_app.banned_tx_ids.add(bytes.fromhex(line))
+            banned_tx_ids.add(bytes.fromhex(line))
+        fp.close()
+
+    if args.ban_addrs:
+        fp = open(args.ban_addrs, 'r')
```

Comment:
Why don't we move it to the `FileFilter`? Maybe add a method `load_from_file(path: str)`?

Reply:
Done.

Reply:
Thanks!